src/ray/core_worker/transport/actor_scheduling_queue.cc

```diff
@@ -35,8 +36,8 @@ ActorSchedulingQueue::ActorSchedulingQueue(
       ss << "\t" << concurrency_group.name << " : " << concurrency_group.max_concurrency;
     }
     RAY_LOG(INFO) << ss.str();
-    fiber_state_manager_ =
-        std::make_unique<FiberStateManager>(concurrency_groups, fiber_max_concurrency);
+    fiber_state_manager_ = std::make_unique<ConcurrencyGroupManager<FiberState>>(
```

Comment:
I feel like `ConcurrencyGroupManager` is a misleading name (since concurrency group is an optional thing here). 

What about `ConcurrentExecutorManager`?

Reply:
Both are fine to me.

Reply:
I'd prefer `ConcurrencyGroupExecutorManager`, which indicates the executor is used for concurrency groups.

Reply:
Hmm but it is also not used by concurrency group right? Like basic async actor or threaded actor

Reply:
Actually, both async actor  and threaded actor are the actors with one default concurrency group.
Ideally all actors can be treated as concurrency group.

Reply:
Hmm I see. In this case, I think it makes sense. (ConcurrencyGroupManager). It might be good to comment what you told me somewhere (that threaded actors o async actor is a concurrency group with default 1 )